src/content/release-notes.mdx

```diff
@@ -5,6 +5,39 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.0
+
+12 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
+
+### New Features
+
+- Added a [feature flag to enable/disable autowake](self-hosted/helm-configuration.mdx#autowake) for Namespaces on incoming request  <!-- 8379 -->
+- Support for GitLab Subgroups and Groups with nested Subgroups <!-- 8398, 8393, 8394 -->
```

Comment:
Good suggestion. I'll update this

Reply:
Adding this to improvements. Should we also keep a `Support for GitLab Groups and Subgroups` callout in the new features line? This feels like it's worth a bigger announcement to me

Reply:
Well, technically we were already supporting it. I mean, you could specify one repository using Groups and Subgroups and it was able to clone it and deploy what was in there. The only thing is that it wasn't picking the right name, so in some specific scenarios, the visualization of the dev environments was mixed, but they were working.

So, that's why I think it would be more an improvement or fix, than a new feature